src/content/docs/en/core-concepts/astro-components.mdx

```diff
@@ -4,11 +4,7 @@ description: An intro to the .astro component syntax.
 i18nReady: true
 ---
 
-**Astro components** are the basic building blocks of any Astro project. They are HTML-only templating components with no client-side runtime.
-
-**If you know HTML, you already know enough to write your first Astro component.**
-
-Astro component syntax is a superset of HTML. The syntax was [designed to feel familiar to anyone with experience writing HTML or JSX](#differences-between-astro-and-jsx), and adds support for including components and JavaScript expressions. You can spot an Astro component by its file extension: `.astro`.
+**Astro components** are the basic building blocks of any Astro project. They are HTML-only templating components with no client-side runtime. You can spot an Astro component by its file extension: `.astro`.
```

Comment:
I know this wasn't introduced in this PR, but this sentence is misleading, as:
- "HTML-only" might sound like we're excluding CSS and JS, which can both be included in an Astro component
- You can include script tags for a client-side runtime

First thought:
```suggestion
**Astro components** are the basic building blocks of any Astro project. They are templating components that are built ahead of time, rather than on the client. You can spot an Astro component by its file extension: `.astro`.
```